docs/source/conf.py

```diff
@@ -319,7 +340,7 @@
 # The name of an image file (relative to this directory) to place at the top
 # of the sidebar.
 #
-html_logo = "_static/arrow.png"
+# html_logo = "_static/arrow.png"
```

Comment:
I have been following the idea of commenting out default settings of the theme so that it is easy to see what can be changed. At least that is what I am seeing in the `docs/source/conf.py`. If I am mistaken, I am happy to remove it.

Some examples of commented code:
- https://github.com/apache/arrow/blob/2a56d4598a3a401447c91645448853b940e86fbe/docs/source/conf.py#L106-L108
- https://github.com/apache/arrow/blob/2a56d4598a3a401447c91645448853b940e86fbe/docs/source/conf.py#L315-L317
- https://github.com/apache/arrow/blob/2a56d4598a3a401447c91645448853b940e86fbe/docs/source/conf.py#L366-L420

Reply:
Ah good to know! I'm okay with leaving it as-is.